src/coalesce-vue-vuetify3/src/components/display/c-table.vue

```diff
@@ -92,10 +105,51 @@ export default defineComponent({
     props: { required: false, type: Array as PropType<Array<string>> },
     admin: { required: false, type: Boolean },
     editable: { required: false, type: Boolean },
-    extraHeaders: { required: false, type: Array as PropType<Array<string>> },
+    extraHeaders: {
+      required: false,
+      type: Array as
+        | PropType<Array<{ header: string; isFixed: boolean }>>
+        | PropType<Array<string>>,
+    },
     loaders: { required: false, type: Object },
   },
 
+  setup() {
+    const cTable = ref<HTMLDivElement | null>(null);
+    const isHorizontalScrollbarVisible = ref(false);
+
+    const checkHorizontalScrollbar = () => {
+      const divElement = cTable.value;
+      const tableElement = cTable.value?.querySelector("table");
+      if (tableElement && divElement) {
+        isHorizontalScrollbarVisible.value =
+          divElement.clientWidth < tableElement.clientWidth;
+      }
+    };
+
+    const resizeObserver = new ResizeObserver(() => {
+      checkHorizontalScrollbar();
+    });
+
+    onMounted(async () => {
+      if (cTable.value) {
+        resizeObserver.observe(cTable.value);
+      }
+      checkHorizontalScrollbar();
+    });
+
+    onUnmounted(() => {
+      if (cTable.value) {
+        resizeObserver.unobserve(cTable.value);
+      }
+    });
```

Comment:
Simpler to just kill the whole observer

```suggestion
    onUnmounted(() => {
      resizeObserver.disconnect();
    });
```